packages/driver/test/cypress/integration/commands/actions/click_spec.js

```diff
@@ -965,6 +965,10 @@ describe('src/cy/commands/actions/click', () => {
         cy.get('#overflow-link').find('.wrapped').click()
       })
 
+      it('can click on inline elements that wrap lines where the first rect has no width', () => {
```

Comment:
can you add the issue link

Reply:
I've reopened https://github.com/cypress-io/cypress/issues/7343. However, I still feel that this may also fix #7012, which might be worth investigating before merging (and then forgetting about it).